chainer/functions/loss/softmax_cross_entropy.py

```diff
@@ -48,14 +62,15 @@ def forward_cpu(self, inputs):
         if count == 0:
             return numpy.zeros((), dtype=x.dtype),
 
-        y = (numpy.log(p) * (t.flat != self.ignore_label)).sum(keepdims=True) \
+        y = (log_p * (t.flat != self.ignore_label)).sum(keepdims=True) \
             * (-1.0 / count)
```

Comment:
I think that type of `count` might be `ndarray`.
https://github.com/pfnet/chainer/pull/712#discussion-diff-46940915R57
What do you think?